src/main/java/com/spotinst/sdkjava/model/responses/elastigroup/azure/v3/ElastigroupApiUpdateCapacityResponseAzure.java

```diff
@@ -0,0 +1,7 @@
+package com.spotinst.sdkjava.model.responses.elastigroup.azure.v3;
+
+import com.spotinst.sdkjava.client.response.BaseServiceItemsResponse;
+import com.spotinst.sdkjava.model.api.azure.elastiGroup.V3.ApiUpdateCapacityAzure;
+
+public class ElastigroupApiUpdateCapacityResponseAzure extends BaseServiceItemsResponse<ApiUpdateCapacityAzure> {
```

Comment:
Remove Elastigroup from naming convention for all response classes.

Reply:
changed